amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/records/UpdateRecordsTests.java

```diff
@@ -95,7 +96,7 @@ public void preconditionUpdateRecordsTests()
     {
         rootCategory = createRootCategory(getRandomName("CATEGORY NAME"));
         unfiledRecordFolder = createUnfiledContainerChild(UNFILED_RECORDS_CONTAINER_ALIAS,
-                getRandomName("UnfiledRecordFolder"), UNFILED_RECORD_FOLDER_TYPE);
+            getRandomName("UnfiledRecordFolder"), UNFILED_RECORD_FOLDER_TYPE);
```

Comment:
We can revert this, unnecessary change / not in line with the rest of the formatting for the method.